src/scene/layer-composition.js

```diff
@@ -439,6 +488,62 @@ class LayerComposition extends EventHandler {
         return result;
     }
 
+    // executes when post-processing camera's render actions were created to propage rendering to
+    // render targets to previous camera as needed
+    propagateRenderTarget(startIndex, renderTarget) {
+        // let lastFixed = null;
+        // let lastFixedClearColor = false;
+        for (let a = startIndex; a >= 0; a--) {
+
+            let ra = this._renderActions[a];
+            const layer = this.layerList[ra.layerIndex];
+
+            // if we hit render action with a render target (other than depth layer), that marks the end of camera stack
+            // TODO: refactor this as part of depth layer refactoring
+            if (ra.renderTarget && layer.id !== LAYERID_DEPTH) {
+                break;
+            }
+
+            // skip over depth layer
+            if (layer.id === LAYERID_DEPTH) {
+                continue;
+            }
+
+            // render it to render target
+            ra.renderTarget = renderTarget;
+        }
+    }
+
+    // logs render action and their properties
+    _logRenderActions() {
```

Comment:
I can see how this might be useful to develop this feature. But do we really want to execute this code even in a debug build? The debug build is really for logging warnings and errors.